### ISSUE TITLE
NVDA + Bootstrap Modal + Chrome: doesn't read elements inside modal with transition effect

### ISSUE THREAD

**Issue by ReWWeR** [nvaccess/nvda#6181]
Steps to reproduce (Chrome):
1. Open the Bootstrap Modals example on the bootstrap website here: http://getbootstrap.com/javascript/#live-demo
2. Click on the "Launch demo modal" button.
3. Try to mouseover inside modal window. It will read content behind the modal and sometimes the content from modal but from not correct place.

If you change size of the window or just remove transition property in CSS it start to read it correctly.

**Comment by stanlaktionov** [comment]
Any updates on this issue?

**Comment by ehollig** [comment]
CC @feerrenrut

**Comment by feerrenrut** [comment]
Using Chrome Version `61.0.3163.100` on Windows 10 with `next-14463,7dca66c4`

I found that the modal dialog was mostly accessible and accurate with the mouse, with the exception of a small percentage of the bottom of the window. I tried this at several zoom levels and found that regardless it was about 20% of the bottom of the window that would not report the modal popup. 

1. I turned on mouse tracking.
1. Browsed to https://getbootstrap.com/docs/3.3/javascript/#live-demo and activated the button 'launch demo modal'
1. Moved the mouse over the elements in the modal all the way to the bottom.

I tried this at several magnification levels (using ctrl + to set) and with several different window sizes.

**Comment by Adriani90** [comment]
@ReWWeR do you see any improvements in NVDA 2019.1 Beta 1 regarding this issue? Thanks.

**Comment by Adriani90** [comment]
This is not reproducible anymore with NVDA alpha-27818,fa89f058 (2023.2.0.27818) and Chrome 111. Closing as works for me.
